addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -568,36 +574,38 @@
                             </div>
                         </div>
                         <group name="product_details">
-                            <group string="General Information">
+                            <group>
                                 <field name="active" invisible="1"/>
                                 <field name="type" invisible="1"/>
-                                <field name="standard_price" class="w-25"
-                                       help="When the cost of an expense product is different than 0, then the user
-                                        using this product won't be able to change the amount of the expense,
-                                        only the quantity. Use a cost different than 0 for expense categories funded by
-                                        the company at fixed cost like allowances for mileage, per diem, accommodation
-                                        or meal."/>
-                                <field name="uom_id" class="w-25" groups="uom.group_uom" options="{'no_create': True}"/>
-                                <field name="uom_po_id" invisible="1"/>
-                                <label for="default_code"/>
-                                <div>
-                                    <field name="default_code" class="w-50"/>
-                                    <span class="d-inline-block">
-                                        <i class="text-muted">Use this reference as a subject prefix when submitting by email.</i>
+                                <field name="currency_id" invisible="1"/>
+                                <label for="standard_price"/>
+                                <div class="o_row" groups="base.group_user">
+                                    <field name="standard_price"
+                                        class="w-50"
+                                        widget="monetary"
+                                        options="{'currency_field': 'currency_id'}"
+                                        help="When the cost of an expense product is different than 0, then the user
+                                            using this product won't be able to change the amount of the expense,
+                                            only the quantity. Use a cost different than 0 for expense categories funded by
+                                            the company at fixed cost like allowances for mileage, per diem, accommodation
+                                            or meal."/>
+                                    <span class="d-flex w-50 ms-2" invisible="standard_price == 0">
+                                        per <field name="uom_id" class="ms-2" options="{'no_create': True}"/>
                                     </span>
                                 </div>
-                                <field name="categ_id" class="w-50"/>
+                                <field name="uom_po_id" invisible="1"/>
```

Comment:
is it still needed ?

Reply:
not in the current code, but maybe removing it would break some inheritance?